pkg/kubelet/images/types.go

```diff
@@ -48,7 +48,7 @@ var (
 // Implementations are expected to be thread safe.
 type ImageManager interface {
 	// EnsureImageExists ensures that image specified in `container` exists.
-	EnsureImageExists(ctx context.Context, pod *v1.Pod, container *v1.Container, pullSecrets []v1.Secret, podSandboxConfig *runtimeapi.PodSandboxConfig, podRuntimeHandler string) (string, string, error)
+	EnsureImageExists(ctx context.Context, ref *v1.ObjectReference, pod *v1.Pod, reference string, pullSecrets []v1.Secret, podSandboxConfig *runtimeapi.PodSandboxConfig, podRuntimeHandler string, pullPolicy v1.PullPolicy) (string, string, error)
```

Comment:
hm can you describe the difference between `ref` and `reference` in a comment? potentially, it may be clearer to rename

Reply:
Yes makes sense, renamed them to become `objRef` and `imgRef`.